### DISCUSSION TITLE
Untangle

### DISCUSSION THREAD

**Discussion by giscus** [aduros/wasm4#771] [Games]
# Untangle

Build retro games using WebAssembly for a fantasy console

https://wasm4.org/play/untangle/

**Comment by 915862621** [comment]
What a good game, this is insane. It's literally my game of the year👏🏻👏🏻

**Comment by gbraad** [comment]
Wish there was more; 24 points is the max. "Thank you for making the game"

**Comment by tandej** [comment]
lovely!
